terraform/cloudwatch.tf

```diff
@@ -1,14 +1,29 @@
 resource "aws_cloudwatch_event_rule" "everyday-11-pm-est" {
   name                = substr("everyday-11-pm-est${local.name_suffix}", 0, 64)
-  description         = "Fires every five minutes"
+  description         = "Run everyday at 11 pm EST"
   schedule_expression = "cron(0 4 ? * * *)" // -5 to EST
 }
 
+resource "aws_cloudwatch_event_rule" "everyday-3-am-est" {
+  name                = substr("everyday-3-am-est${local.name_suffix}", 0, 64)
+  description         = "Run everyday at 3 am EST"
+  schedule_expression = "cron(0 8 ? * * *)" // -5 to EST
+}
+
 //We only want to schedule this event in production
 resource "aws_cloudwatch_event_target" "nightly-new-area-check" {
   rule      = aws_cloudwatch_event_rule.everyday-11-pm-est.name
   target_id = substr("${local.project}-summarize_new_areas${local.name_suffix}", 0, 64)
   arn       = aws_sfn_state_machine.new_user_aoi.id
   role_arn  = aws_iam_role.datapump_states.arn
-  count     = var.environment == "production" ? 1 : 0
+  count     = var.environment == "production" || var.environment == "staging" ? 1 : 0
+}
+
+//We only want to schedule this event in production
+resource "aws_cloudwatch_event_target" "nightly-new-glad-alerts-check" {
+  rule      = aws_cloudwatch_event_rule.everyday-3-am-est.name
+  target_id = substr("${local.project}-summarize_new_areas${local.name_suffix}", 0, 64)
+  arn       = aws_sfn_state_machine.new_glad_alerts.id
+  role_arn  = aws_iam_role.datapump_states.arn
+  count     = var.environment == "production" || var.environment == "staging" ? 1 : 0
```

Comment:
Good for now. Eventually, we should trigger the update directly after glad alerts were updated.

Reply:
The issue right now is the glad alerts are updating on a different AWS account, so you can't set a trigger (without doing some weird hacky stuff). I figure just do this until glad alerts are updated on this account (although we'll need some kind of "finished" file, since we can't trigger based on individual tiles updating)

Reply:
yes, we should work on moving the update process to the new account. While doing that we should also make use of batch and step functions. We can tackle this in the new year